Exiled.Loader/Loader.cs

```diff
@@ -347,23 +352,22 @@ private static bool CheckPluginRequiredExiledVersion(IPlugin<IConfig> plugin)
             var requiredVersion = plugin.RequiredExiledVersion;
             var actualVersion = Version;
 
-            // Check Major version
-            // It's increased when an incompatible API change was made
-            if (requiredVersion.Major != actualVersion.Major)
+            // Check the version
+            if (requiredVersion != actualVersion)
             {
-                // Assume that if the Required Major version is greater than the Actual Major version,
+                // Assume that if the Required version is greater than the Actual version,
                 // Exiled is outdated
-                if (requiredVersion.Major > actualVersion.Major)
+                if (requiredVersion > actualVersion)
                 {
-                    Log.Error($"You're running an older version of Exiled ({Version.ToString(3)})! {plugin.Name} won't be loaded! " +
-                              $"Required version to load it: {plugin.RequiredExiledVersion.ToString(3)}");
+                    Log.Error($"You're running an older version of Exiled ({Version})! {plugin.Name} won't be loaded! " +
+                              $"Required version to load it: {plugin.RequiredExiledVersion}");
 
                     return true;
                 }
-                else if (requiredVersion.Major < actualVersion.Major && !Config.ShouldLoadOutdatedPlugins)
+                else if (requiredVersion < actualVersion && !Config.ShouldLoadOutdatedPlugins)
```

Comment:
This check can remain a .Major check only, as older plugins requiring older versions of EXILED are *typically* save to load so long as the EXILED major version is the same. The only real issues from mismatched EXILED versions come if the plugin requires a newer version of EXILED than is on the server, as things get added only change the .Minor, if something is removed completely, the .Major version will always have to change.